packages/cloud_firestore/lib/src/query.dart

```diff
@@ -25,9 +25,7 @@ class Query {
   final List<String> _pathComponents;
   final Map<String, dynamic> _parameters;
 
-  /// A string containing the slash-separated path to this Query (relative to
-  /// the root of the database).
-  String get path => _pathComponents.join('/');
+  String get _path => _pathComponents.join('/');
```

Comment:
Breaking change? Anyway, changelog entry and version bump needed.

Reply:
Yeah, it's breaking. Our goal here is to match the iOS and Android SDKs, which put this API on CollectionReference